extensions/notebook/src/extension.ts

```diff
@@ -120,9 +118,9 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
 	}
 
 	let workspaceFolders = vscode.workspace.workspaceFolders?.slice() ?? [];
-	const bookTreeViewProvider = new BookTreeViewProvider(appContext.apiWrapper, workspaceFolders, extensionContext, false, BOOKS_VIEWID);
+	const bookTreeViewProvider = new BookTreeViewProvider(appContext.apiWrapper, workspaceFolders, extensionContext, false, BOOKS_VIEWID, NavigationProviders.NotebooksNavigator);
```

Comment:
this is totally random but this reminds me of a baseball team near where i grew up:

[Norwich naviagators](http://logotypes101.com/logos/435/BCAB94B69EE1DD3FC57A0E1113AB50CF/Norwich_Navigators86.png)